lib/internal/modules/cjs/loader.js

```diff
@@ -989,14 +989,14 @@ Module.prototype.load = function(filename) {
   Module._extensions[extension](this, filename);
   this.loaded = true;
 
-  const ESMLoader = asyncESM.ESMLoader;
+  const esmLoader = asyncESM.esmLoader;
```

Comment:
The reason I made the change in this PR is because the previous code misleads (based on nigh universal convention): that is _not_ a class; it's an insurance. Also, I changed the name of the loader class, which seemed to also bring this in-scope (I myself was misled by the PascalCase when it blew up).

The differentiation also facilitates grepping.

Not a hill to die on, but seems relevant in an atomic-commit kinda way.